Hermes-Marketing/hermes
# Navbar template

## Describe your changes

Created a template for the Navbar. This includes Vue router, icons (from fontAwesome), and basic styling.

## Issue ticket number and link

Ticket #3: Create Category Navbar

## How to test changes

Open website using docker-compose up. Should show Navbar template. 
     If reference error arises for vue-router the following 3 steps should resolve:
          - docker-compose build
          - docker-compose down
          - docker-compose up

Click on Hermes to be routed to home view, and Admin to be routed to Admin view.
To ensure effective routing, check network requests when routing between the two. Should show no new network requests.

## Checklist before requesting a review

- [ Y ] I have performed a self-review of my code
- [ Y ] If it is a core feature, I have added thorough tests.
- [ N ] Do we need to implement analytics?
